playbooks/README.md

```diff
@@ -108,10 +108,13 @@ installed and a config file in ```$HOME/.kube/config```.  Also make sure you
 indentified with the cluster using ```oc login```.
 
 Before installing KubeVirt on an existing OpenShift cluster, ensure that SELinux is disabled on all hosts:
+
 ```bash
 $ ansible-playbook -i inventory playbooks/selinux.yml
```

Comment:
here you reference your inventory sample, probably better to note that this is a sample file that may be used (after being modified with the right node info), also, please note that if the user don't want to use it they may take their own inventory (otherwise, someone may just copy&paste this line as-is).

Reply:
@yuvalif Will it be ok if I chage line 116 for: 

`Be sure to update the [inventory file](../inventory) according to your OpenSift cluster configuration or use the file you used to deploy the cluster.`